packages/duppy-windows.0.7.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "Romain Beauxis <toots@rastageeks.org>"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+homepage: "https://github.com/savonet/ocaml-duppy"
+build: [
+  ["./configure" "--host=%{conf-gcc-windows:host}%" "--prefix=%{prefix}%" "OCAMLFIND_TOOLCHAIN=windows"]
+  [make]
+]
+install: [
+  ["env" "OCAMLFIND_TOOLCHAIN=windows"
+   "make" "install"]
+]
+remove: ["ocamlfind" "-toolchain" "windows" "remove" "duppy"]
+depends: [
+  "ocamlfind" {build}
```

Comment:
Missing `"ocaml-windows"` dep; intentional or not? I prefer every package to specify it explicitly, since with optional dependencies and changes over time it's not always clear what the true dependency edges are...

Reply:
Mistake..

Reply:
Wait no, sorry not a mistake. I usually add `ocaml-windows` when there isn't another `-windows` dependency. Here, `pcre-windows` already pulls `ocaml-windows`. Do you think we should rather add `ocaml-windows` to all `-windows` packages?

Reply:
I think we should, yes, precisely because it's more obvious.

Reply:
Word.